### PR TITLE
Add customizable footer on login screen

### LINKED ISSUE #11: Add footer to login page with dynamic text
A footer should be added to the login page.

This can be used as reference:
image

We don't need the social-media icons.

The logo should be a black and white version of our logo:

[Traefik Guardian Logo.zip](https://github.com/MarvinJWendt/traefik-guardian/files/9705275/Traefik.Guardian.Logo.zip)

The copyright text should be modifiable via an environment variable (injected via go template).

### PR DESCRIPTION
**Pull request by Aragur**
This PR implements a customizable footer on the login screen.
This can be customized via the `LOGIN_PAGE_COPYRIGHT` environment variable.

Resolves #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,12 +61,13 @@ You can find a full example, including Traefik and a demo service, here: [_examp
...
+| Variable Name          | Description                                | Accepted Values                                 | Default Value                         |
+|------------------------|--------------------------------------------|-------------------------------------------------|---------------------------------------|
+| `AUTH_HOST`            | The host to use.                           | Any valid host (e.g.: `auth.example.com`)       |                                       |
+| `PASSWORDS`            | The passwords that can be used to login.   | See [Password Management](#password-management) |                                       |
+| `DEBUG`                | Enable debug mode.                         | `true`, `false`                                 | `false`                               |
+| `LOGIN_PAGE_TITLE`     | Title of the login page.                   | Any string.                                     | `Traefik Guardian - Login`            |
+| `LOGIN_PAGE_COPYRIGHT` | Copyright at the bottom of the login page. | Any string.                                     | `Copyright © 2022 - Traefik Guardian` |
```
Comment (MarvinJWendt) [inline README.md:70]: Please rename the env. variable to `LOGIN_PAGE_FOOTER_TEXT`, since it doesn't need to be a copyright text.

**Review by MarvinJWendt — CHANGES_REQUESTED** [review]
Thanks for your PR! I have noted a small change. Please resolve that and the PR will be ready to merge :)

File: README.md (hunk 1 above)
Comment (Aragur) [reply README.md]: That's a better name for this variable. Thanks 👍

**Review by MarvinJWendt — APPROVED** [review]
LGTM! Thanks for contributing! 🚀

**Pull request merged**